doc/source/ray-air/examples/gptj_batch_prediction.ipynb

```diff
@@ -63,6 +63,7 @@
     "        \"pip\": [\n",
     "            \"accelerate>=0.16.0\",\n",
     "            \"transformers>=4.26.0\",\n",
+    "            \"numpy<1.24\",\n",
```

Comment:
can you comment here that latest numpy releases >= 1.24 breaks most recent transformers release.
so we know when to get rid of the pin.